edu.stanford.protege.appdata.xml

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0"?>
+<component>
+    <id>edu.stanford.protege</id>
+    <name>Protégé</name>
+    <summary>
+        Protégé is supported by a strong community of academic, government, and
+        corporate users, who use Protégé to build knowledge-based solutions in
+        areas as diverse as biomedicine, e-commerce, and organizational
+        modeling.
+    </summary>
+    <url type="homepage">https://pprotege.stanford.edu/</url>
+    <metadata_license>BSD-3-Clause</metadata_license>
```

Comment:
preferred metadata license (it's jus this file) is `CC0-1.0`. Not even sure `BSD-3-Clause` is acceptable.
Maybe you are confusing with the `project_license` that is missing.

Missing mandatory `releases`, `content_rating`, `screenshots`, at least

Reply:
You are right, this is the project's license. I'll switch it to CC.